yarn-project/archiver/src/archiver/l1/calldata_retriever.test.ts

```diff
@@ -43,8 +52,13 @@ class TestCalldataRetriever extends CalldataRetriever {
     return await super.extractCalldataViaTrace(txHash);
   }
 
-  public override decodeAndBuildCheckpoint(proposeCalldata: Hex, blockHash: Hex, checkpointNumber: CheckpointNumber) {
-    return super.decodeAndBuildCheckpoint(proposeCalldata, blockHash, checkpointNumber);
+  public override decodeAndBuildCheckpoint(
+    proposeCalldata: Hex,
+    blockHash: Hex,
+    checkpointNumber: CheckpointNumber,
+    expectedHashes?: { attestationsHash?: Hex; payloadDigest?: Hex },
```

Comment:
```suggestion
    expectedHashes: { attestationsHash?: Hex; payloadDigest?: Hex },
```
Nit, so we don't accidentally forget passing this argument in a callsite